go.mod

```diff
@@ -70,7 +70,7 @@ require (
 	github.com/openshift/api v0.0.0-20211028023115-7224b732cc14
 	github.com/openshift/apiserver-library-go v0.0.0-20211105091019-06e87e7030eb
 	github.com/openshift/client-go v0.0.0-20210831095141-e19a065e79f7
-	github.com/openshift/library-go v0.0.0-20211207082558-36821c7a1fa2
+	github.com/openshift/library-go v0.0.0-20220405121559-e304504b7d6f
```

Comment:
okay, this points to https://github.com/openshift/library-go/commits/release-4.9